keyboards/skyloong/gk980/bl/info.json

```
@@ -0,0 +1,55 @@
+{
+    "manufacturer": "skyloong",
+    "maintainer": "NaturalZh",
+    "bootloader": "stm32duino",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A1", "pin_b": "A0", "resolution": 2},
+            {"pin_a": "A3", "pin_b": "A2", "resolution": 2},
+            {"pin_a": "A4", "pin_b": "B0", "resolution": 2},
+            {"pin_a": "B3", "pin_b": "B1", "resolution": 2},
+            {"pin_a": "B10", "pin_b": "B4", "resolution": 2}
+        ]
+    },
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "command": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "C15",
+        "on_state": 1,
+        "scroll_lock": "C14"
+    },
+    "matrix_pins": {
+        "cols": [null, null, null, null, null, null, null, null, null, null, null, null, null, null, null],
+        "rows": ["A8", "A9", "A10", "A15", "B6", "B7","B9"]
+    },
+    "processor": "STM32F103",
+    "backlight": {
+        "breathing": true,
+        "breathing_period": 5,
+        "default": {
+            "breathing": false,
+            "brightness": 30
```

Cannot be higher than number of levels
```suggestion
            "breathing": false
```